apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryTable.cs

```diff
@@ -161,6 +161,28 @@ internal BigQueryTable(BigQueryClient client, Table resource)
         public void Delete(DeleteTableOptions options = null) => _client.DeleteTable(Reference, options);
 
         /// <summary>
+        /// Asynchronously creates a job to extract data from this table to Google Cloud Storage.
+        /// This method just creates a <see cref="TableReference"/> and delegates to <see cref="BigQueryClient.CreateExtractJob(TableReference, String, CreateExtractJobOptions)"/>.
+        /// See [the BigQuery documentation](https://cloud.google.com/bigquery/docs/exporting-data) for more information on extract jobs.
+        /// </summary>
+        /// <param name="destinationUri">The Google Cloud Storage URI (possibly including a wildcard) to extract the data to. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <returns>The job created for the extract operation.</returns>
+        public BigQueryJob CreateExtractJob(string destinationUri, CreateExtractJobOptions options = null) =>
+            _client.CreateExtractJob(Reference, destinationUri, options);
+
+        /// <summary>
+        /// Asynchronously creates a job to extract data from this table to Google Cloud Storage.
```

Comment:
Gah. Apologies for these. The joys of mind-numbing C&P in the name of usability. (All the overloads and different entry points are definitely worth it for usability, but getting all the ducks in a row is a pain.)